install/kubernetes/helm/istio/values-istio.yaml

```diff
@@ -8,4 +8,4 @@ global:
   # Default is 10s second
   refreshInterval: 1s
   proxy:
-    image: proxy
\ No newline at end of file
+    image: proxyv2
```

Comment:
I found only by changing this to proxyv2 the sidecar can use the new v2 api (where my changes locates) to get the configuration. I don't want to change this value-istio.yaml  globally. but now to make my test only targeted to proxyv2 image? Should it be `if !tc.v1alpha3 { t.Skip("" }` ?

The command I uses to run the tests:
```
make e2e_pilot E2E_ARGS='--auth_enable=false -run=".*Multiplex.*" --skip_cleanup=true --skip_setup=false --namespace=authn-e2e HUB="gcr.io/jianfeih-test" --v1alpha1=false --v1alpha3=true'
```

Reply:
I thought we switched globally to v2 proxy by default. @linsun / @gyliu513 ??

Reply:
After trying out the correct test target, `make e2e_pilotv2_v1alpha3`, this seems no longer needed.